src/app/core/rating-module/rating.service.ts

```diff
@@ -251,31 +210,23 @@ export class RatingService {
   /**
    * Flag a rating
    *
-   * @param {string} CUID
-   * @param {string} version
    * @param {string} ratingId
    * @param {{concern: string, comment: string}} report
    * @returns {Promise<any>}
    */
   flagLearningObjectRating(params: {
-    username: string;
-    CUID: string;
-    version: number;
     ratingId: string;
     report: { concern: string, comment?: string }
```

Comment:
Comment isn't used in the backend.

```suggestion
    report: { concern: string }
```